features/features.go

```diff
@@ -166,10 +168,26 @@ var featuresSupportedCallbacks = map[SnapdFeature]func() (bool, string){
 		}
 		return true, ""
 	},
-	// AppArmorPrompting requires a newer version of snapd with all the
+	// AppArmorPrompting requires AppArmor parser and kernel support for
+	// prompting, as well as a newer version of snapd with all the
 	// prompting components in place. TODO: change this callback once ready.
 	AppArmorPrompting: func() (bool, string) {
+		kernelFeatures, err := apparmor.KernelFeatures()
+		if err != nil {
+			return false, fmt.Sprintf("error checking apparmor kernel features: %v", err)
+		}
+		if !strutil.ListContains(kernelFeatures, "policy:permstable:prompt") {
```

Comment:
I'm confused, the other package has permstable32, not just permstable:

https://github.com/snapcore/snapd/pull/13821/files#diff-3358a00cc3be160b5e919542e6232b3ff02a044848a99c55d3b34df2406664adR372

am I missing something?

Reply:
hm I suppose this was caused by splitting it in 2 PRs, one advancing some more than the other, https://github.com/snapcore/snapd/pull/13822

Reply:
Yes, this was a typo on my part when splitting out the PR, this should be `"policy:permstable32:prompt"`. Thanks for catching this